src/lib/hooks/useStopPlaceName.ts

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from 'react'
+import { getTripLocations } from '@/lib/api/eventApi'
+import { useEventName } from './useEventName'
+
+export const useStopPlaceNames = () => {
+    const [startLocationName, setStartLocationName] = useState<string | null>(
+        null,
+    )
+    const [endLocationName, setEndLocationName] = useState<string | null>(null)
+    const [error, setError] = useState<string | null>(null)
+    const { eventName } = useEventName()
+
+    useEffect(() => {
```

Comment:
Kunne fungert med en useSWR her og for å få gratis error, men ikke nødvendig om man ikke ønsker